src/oauth.js

```diff
@@ -49,7 +48,10 @@ passport.use(new Strategy(
         })
       }
     })
-    .catch(error => { throw new Error(error.message) });
+    .catch(error => { 
+      console.log(error);
```

Comment:
To keep it consistent, maybe change this and line 68 to `console.log(err.message)`